server/routes/api/index.js

```diff
@@ -3,6 +3,14 @@ const express = require('express');
 const router = express.Router();
 const snipApi = require('./snip');
 
+if (express().get('env') == 'development') {
+  router.use(function(req, res, next) {
```

Comment:
config file has a variable called env Please use it wherever we need to check env.
Also better to use cors library
https://www.npmjs.com/package/cors
use as app.use('/api', cors()) in index.js itself
It is easier and supports more options which we may need in future

Reply:
👍